### BIG differences, this is diff of wrong files.
### CLUS "System Class CONS"
### and CLHS topic selection http://clhs.lisp.se/Body/a_cons.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:types:cons.txt	2017-04-27 22:01:07.000000000 +0100
+++ saved-pages-clus/cl:types:cons.txt	2017-04-27 22:01:07.000000000 +0100
@@ -1,4 +1,43 @@
-CONS
-Please select which reference to CONS you intended
+System Class CONS

-FunctionSystem ClassGlossary Entry
\ No newline at end of file
+Class Precedence List
+cons, list, sequence, t
+Description
+A cons is a compound object having two components, called the car and cdr. These form a dotted pair. Each component can be any object.
+Compound Type Specifier Kind
+Specializing.
+Compound Type Specifier Syntax
+cons [car-typespec [cdr-typespec]]
+Compound Type Specifier Arguments
+car-typespec---a type specifier, or the symbol *. The default is the symbol *.
+cdr-typespec---a type specifier, or the symbol *. The default is the symbol *.
+Compound Type Specifier Description
+This denotes the set of conses whose car is constrained to be of type car-typespec and whose cdr is constrained to be of type cdr-typespec. (If either car-typespec or cdr-typespec is *, it is as if the type t had been denoted.)
+====See Also====
+{\secref\LeftParen}, {\secref\PrintingListsAndConses}
+\
\ No newline at end of file
